README.md

```diff
@@ -13,7 +13,7 @@ Can't wait to generate some fake data ? Follow the steps:
   
 ```elixir
 def deps do
-  [{:faker_elixir_octopus, "> 0.0.0",  only: [:dev, :test]}]
+  [{:faker_elixir_octopus, "~>0.13.1",  only: [:dev, :test]}]
```

Comment:
Okey, I will create a MR later and add some extra badges on top of the readme with the current version as well.